src/elements/content-sidebar/activity-feed/task-new/Assignees.js

```diff
@@ -73,12 +73,7 @@ class Assignees extends React.Component<Props> {
                             />
                         }
                     >
-                        <AssigneeStatus
```

Comment:
for whatever reason, the `data-testid` in the `wrapper.find` was returning both this top level function and the nested `div` causing 2x the number of objects returned. moving the data-testid into the child component fixes that without impacting usage